tests/describe_task_queue_test.go

```diff
@@ -140,34 +114,14 @@ func (s *DescribeTaskQueueSuite) PublishConsumeWorkflowTasksValidateBacklogInfo(
 			Identity:            identity,
 		}
 
-		resp0, err0 := s.engine.StartWorkflowExecution(NewContext(), request)
+		_, err0 := s.engine.StartWorkflowExecution(NewContext(), request)
 		s.NoError(err0)
-
-		we := &commonpb.WorkflowExecution{
-			WorkflowId: id,
-			RunId:      resp0.RunId,
-		}
-
-		s.EqualHistoryEvents(`
-  1 WorkflowExecutionStarted
-  2 WorkflowTaskScheduled`, s.getHistory(s.namespace, we))
 	}
 
-	response := s.ValidateDescribeTaskQueue(tl)
-	versionInfo := response.GetVersionsInfo()[""]
-
-	// verifying backlog counter increased as tasks were added
-	addedTasks := int64(workflows)
-	for qT, t := range versionInfo.GetTypesInfo() {
-		queueType := enumspb.TaskQueueType(qT)
-		if queueType == enumspb.TASK_QUEUE_TYPE_WORKFLOW {
-			s.Assert().Equal(t.BacklogInfo.ApproximateBacklogCount, addedTasks)
-		} else if queueType == enumspb.TASK_QUEUE_TYPE_ACTIVITY {
-			s.Assert().Equal(t.BacklogInfo.ApproximateBacklogCount, int64(0))
-		}
-	}
+	expectedBacklogCount[enumspb.TASK_QUEUE_TYPE_WORKFLOW] = int64(workflows)
+	s.validateDescribeTaskQueue(tl, expectedBacklogCount)
 
-	// starting and completing the workflow tasks by fetching their workflowExecutions
+	// Completing the tasks
```

Comment:
we're only polling the task which takes it out of backlog, I think "completing" would be confusing as it can mean performing the polled task by a worker.

```suggestion
	// Polling the tasks
```